recipes/arrow/all/conanfile.py

```diff
@@ -237,25 +237,25 @@ def requirements(self):
         if self._with_thrift():
             self.requires("thrift/0.15.0")
         if self._with_protobuf():
-            self.requires("protobuf/3.19.2")
+            self.requires("protobuf/3.20.0")
         if self._with_jemalloc():
             self.requires("jemalloc/5.2.1")
         if self._with_boost():
-            self.requires("boost/1.78.0")
+            self.requires("boost/1.79.0")
         if self._with_gflags():
             self.requires("gflags/2.2.2")
         if self._with_glog():
-            self.requires("glog/0.5.0")
+            self.requires("glog/0.6.0")
         if self._with_grpc():
-            self.requires("grpc/1.44.0")
+            self.requires("grpc/1.45.2")
         if self.options.with_json:
             self.requires("rapidjson/1.1.0")
         if self._with_llvm():
             self.requires("llvm-core/13.0.0")
         if self._with_openssl():
-            self.requires("openssl/1.1.1m")
+            self.requires("openssl/1.1.1o")
```

Comment:
When will we do the jump to 3?
Linux distributions are beginning to make the jump.

Reply:
@madebr 
I agree with you. I will try.

Reply:
Fixed.
openssl/3.0.3 works well in my environment.

Reply:
I fear this will cause version conflicts as with the bump from zlib/1.2.11 to zlib/1.2.12.
But the upgrade to 3.0 should happen at some point.